spec/article_json/export/apple_news/elements/base_spec.rb

```diff
@@ -0,0 +1,75 @@
+describe ArticleJSON::Export::AppleNews::Elements::Base do
+  subject(:element) { described_class.new(source_element) }
+
+  describe '#export' do
+    subject { element.export }
+
+    let(:sample_text) { ArticleJSON::Elements::Text.new(content: 'Foo Bar') }
+    let(:sample_paragraph) do
+      ArticleJSON::Elements::Paragraph.new(content: [sample_text])
+    end
+
+    context 'when the source element is a text' do
+      let(:source_element) { sample_text }
+      it { should eq 'Foo Bar' }
+    end
+
+    context 'when the source element is a heading' do
+      let(:source_element) do
+        ArticleJSON::Elements::Heading.new(content: 'Foo Bar', level: 1)
+      end
+      let(:heading) { { role: 'heading1', text: 'Foo Bar' } }
+      it { should eq heading }
+    end
+
+    context 'when the source element is a paragraph' do
+      let(:source_element) { sample_paragraph }
```

Comment:
I think the `sample_paragraph` above can be removed if you specify it directly in this context:
```suggestion
      let(:source_element) { ArticleJSON::Elements::Paragraph.new(content: [sample_text]) }
```